recipes/itk-dev/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "4.11.0" %}
+
+package:
+  name: itk-dev
+  version: "{{version}}"
+
+source:
+  git_url: git://itk.org/ITK.git
+  git_tag:  v{{version}}
+
+build:
+    number: 0
+
+requirements:
+  build:
+    - cmake    >=3.3
+    - ninja
+    - libtiff  4.0.*
+    - libpng   >=1.6.27,<1.7
+    - jpeg     9*
+    - zlib     1.2.*
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+  run:
+    - libtiff  4.0.*
+    - libpng   >=1.6.27,<1.7
+    - jpeg     9*
+    - zlib     1.2.*
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+
+test:
+  commands:
+    - echo "Testing..."
+
+about:
+  home: http::/www.itk.org
+  license: Apache 2.0
```

Comment:
So I guess pick one and add it with [`license_file`]( https://conda.io/docs/building/meta-yaml.html#license-file ). Would guess `LICENSE` would be the one to go with, but leaving it up to you.